### PR TITLE
doc: Document ZVM backend variables

### PR DESCRIPTION
**Pull request by kalikiana**

### CODE REVIEW

**Review by okurz — CHANGES_REQUESTED** [review]
We should only include variables that our check flags. We need https://github.com/os-autoinst/os-autoinst/pull/1912 for which we likely need this change.

**Comment by kalikiana** [comment]
> We should only include variables that our check flags. We need #1912 for which we likely need this change.

If this branch needs to be merged first, what other changes did you have in mind?

**Comment by okurz** [comment]
Following TDD I want to make sure that we actually need the changes to fix the check after correcting the variable use in backends and consoles

**Comment by okurz** [comment]
https://github.com/os-autoinst/os-autoinst/pull/1912 includes your change

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -43,6 +43,16 @@ UPLOAD_INACTIVITY_TIMEOUT;integer;300;Specifies the inactivity timeout in second
 
 |====================
 
+.ZVM backend
+[grid="rows",format="csv"]
+[options="header",cols="^m,^m,^m,v",separator=";"]
+|====================
+Variable;Values allowed;Default value;Explanation
+ZVM_HOST;string;;Sets the remote host name.
```
Comment (Martchus) [inline doc/backend_vars.asciidoc:51]:
```suggestion
ZVM_HOST;string;;Sets the remote hostname.
```

File: doc/backend_vars.asciidoc [hunk 2]
```diff
@@ -43,6 +43,16 @@ UPLOAD_INACTIVITY_TIMEOUT;integer;300;Specifies the inactivity timeout in second
...
+ZVM_GUEST;string;;Sets the remote user name.
```
Comment (Martchus) [inline doc/backend_vars.asciidoc:52]:
```suggestion
ZVM_GUEST;string;;Sets the remote username.
```
(Just for consistency with your other PR.)

**Review by Martchus — CHANGES_REQUESTED** [review]

File: doc/backend_vars.asciidoc (hunk 2 above)
Comment (okurz) [reply doc/backend_vars.asciidoc:52]: fixed in https://github.com/os-autoinst/os-autoinst/pull/1912

**Comment by okurz** [comment]
covered by https://github.com/os-autoinst/os-autoinst/pull/1912 where your commit is included

**Pull request closed without merging**